Friends/shannonmcook.html

```diff
@@ -0,0 +1,152 @@
+          display: block;
+          float: right;
+          margin-top: 0;
+        }
+        #header-nav li {
+          display: inline;
+          padding-left: 20px;
+        }
+        .container {
+          max-width: 1000px;
+          margin: 0 auto;
+        }
+        #footer {
+          background-color: #2f2f2f;
+          padding: 50px 0;
+        }
+        #footer h4 {
+          color: white;
+          text-transform: uppercase;
+          letter-spacing: 0.1em;
+        }
+        #footer p {
+          color: white;
+        }
+        .column {
+          min-width: 300px;
+          display: inline-block;
+          vertical-align: top;
+        }
+        a {
+          color: #1abc9c;
+          text-decoration: none;
+        }
+        a:hover {r
+          color: #F6a623;
+        }
+        .post-container:nth-child(even) {
+          background: #e8e8ee;
+        }
+        
+        .post, .about {
+          max-width: 800px;
+          margin: 0 auto;
+          padding: 60px 0;
+        }
+        .post-author img {
+          width: 50px;
+          height: 50px;
+          vertical-align: middle;
+        }
+        .post-author span {
+          margin-left: 16px;
+        }
+        .post-date {
+          color: #D2D2D2;
+          font-size: 14px;
+          font-weight: bold;
+          text-transform: uppercase;
+          letter-spacing: 0.1em;
+        }
+        .about {
+          text-align: center;
+        }
+        .post-author img,
+        .about-author img {
+          border-radius: 50%;
+        }
+        h1, h2, h3, h4 {
+          color: #333;
+        }
+        p {
+          line-height: 1.5;
+        }
+        </style>
+
+
+</head>
+<body>
+  <div id="header">
+    <div class="container">
+      <a id="header-title" href="https://shannonmcook.github.io/"><strong>My Blog</strong></a>
+      <ul id="header-nav">
+        <li><a href="shannonmcook.html">About</a></li>
+        <li><a href="mailto:cook.shannonmarie@gmail.com">Contact</a></li>
+      </ul>
+    </div>
+  </div>
+  <div id="content">
+    <div class="container">
+      <div class="about">
+        <div class="about-author">
+          <img src="https://github.com/shannonmcook/my-blog/blob/4d64b42b78bc87b18ad2b630f77a839c9fcfdd08/me1.png?raw=true">
+        </div>
+        <h1 class"about-title">About Me</h1>
+        <div class="about-content">
+          <p>Oh, hi! I didn't see you there! I am an aspiring web developer.</p>
+          <p>I love computers, puzzles, and a good scavenger hunt. These activities are perfectly compatible with web develpment. If you like my blog, please follow me on <a href="https://twitter.com/shancoo?lang=en">Twitter</a> and tell your family and friends about it. If you want to get in touch, please click <a href="mailto:cook.shannonmarie@gmail.com">here</a> or you can check out my GitHub profile at <a href="https://github.com/shannonmcook">https://github.com/shannonmcook</a>.</p>

```

Comment:
I appreciate the feedback and your time looking over my commits. Initially, I misunderstood that the link had to be in the html file. I added the GitHub link here on the last commit, to line 129 in the paragraph tag:  `<p>I love computers, puzzles, and a good scavenger hunt. These activities are perfectly compatible with web develpment. If you like my blog, please follow me on <a href="https://twitter.com/shancoo?lang=en">Twitter</a> and tell your family and friends about it. If you want to get in touch, please click <a href="mailto:cook.shannonmarie@gmail.com">here</a> or you can check out my GitHub profile at <a href="https://github.com/shannonmcook">https://github.com/shannonmcook</a>.</p>`

I then closed the commits, and resubmitted a new pull request where I added my GitHub link to the header so it would be more easily identifiable right off the bat. Is closing commits and remaking a pull request acceptable, or should I have continued editing the commits from my previous pull request? Thanks again for your patience with me!